admin_guide/out_of_resource_handling.adoc

```diff
@@ -226,7 +226,7 @@ Thresholds are configured in the following form:
 
 * the `eviction-signal` value can be any xref:out-of-resource-eviction-signals-supported[supported eviction signal].
 * the `operator` value is `<`.
-* the `quantity` value must match the link:https://github.com/kubernetes/community/blob/master/contributors/design-proposals/resources.md#resource-quantities[quantity representation] used by
+* the `quantity` value must match the link:https://github.com/kubernetes/community/blob/58b1c30d95719749068497ba35dfe4c64b21aa72/contributors/design-proposals/scheduling/resources.md#resource-quantities[quantity representation] used by
```

Comment:
Also a better URL would be: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/resources.md#resource-quantities